src/main/java/com/training/basepages/FlipkartBasePage.java

```diff
@@ -1,26 +1,18 @@
 package com.training.basepages;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class FlipkartBasePage{
-    protected WebDriver driver;
-    protected WebDriverWait wait;
+import com.codeborne.selenide.SelenideElement;
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.*;
 
-    private By loaderIcon = By.xpath("//div[@class='_2YsvKq _3bgaUQ']/*[name()='svg']");
-    private By cartIcon = By.xpath("//*[text()='Cart']");
-
-    public FlipkartBasePage(WebDriver driver) {
-        this.driver = driver;
-        this.wait = new WebDriverWait(driver, 30);
-    }
+public class FlipkartBasePage {
+    private SelenideElement loaderIcon = $x("//div[@class='_2YsvKq _3bgaUQ']/*[name()='svg']");
+    private SelenideElement cartIcon = $x("//*[text()='Cart']");
```

Comment:
Lets define this in a string rather than SelenideElement, and then use it in your methods